usage/app/lib/UsageRecorder.scala

```diff
@@ -77,19 +86,23 @@ class UsageRecorder(
         val result = func(mediaUsage)
         logger.info(
           logMarker,
-          s"'$opName' DB Operation for ${mediaUsage.grouping} -  on mediaID: ${mediaUsage.mediaId} with result: ${result}"
+          s"'$opName' DB Operation for ${mediaUsage.grouping} - on mediaID: ${mediaUsage.mediaId} with result: $result"
```

Comment:
`result` is not nicely stringified, appearing in the logs as e.g. `rx.lang.scala.JavaConversions$$anon$2@68ffa00f`

Reply:
improved in https://github.com/guardian/grid/pull/3735/commits/13c86e4cc4cf1dd9f00511fb18b776b77dceb7b0